sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -2109,12 +2090,10 @@ class DataFrameSuite extends QueryTest with SharedSQLContext {
   }
 
   test("SPARK-25714 Null handling in BooleanSimplification") {
-    withSQLConf(SQLConf.OPTIMIZER_EXCLUDED_RULES.key -> ConvertToLocalRelation.ruleName) {
-      val df = Seq(("abc", 1), (null, 3)).toDF("col1", "col2")
```

Comment:
Is it ok to remove `SQLConf.OPTIMIZER_EXCLUDED_RULES.key -> ConvertToLocalRelation.ruleName` here?

Reply:
yea, this is the default. See `SharedSparkSession#L48`